AdyenCardScanner/Sources/CardScanner.swift

```diff
@@ -42,4 +43,24 @@ public enum CardScanner: CardScanning {
     public static var isAvailable: Bool {
         cardScannerAssembler.captureDevice != nil
     }
+
+    // MARK: - Private
+
+    // swiftlint:disable line_length
+    private static func assertCameraUsageDescription() {
+        guard let cameraUsageDescription = Bundle.main.object(
+            forInfoDictionaryKey: "NSCameraUsageDescription"
+        ) as? String, !cameraUsageDescription.isEmpty else {
+            #if DEBUG
+                fatalError(
+                    "Error: NSCameraUsageDescription is missing from Info.plist. Please add a description for camera usage as required by the AdyenCardScanner SDK."
+                )
+            #else
+                debugPrint(
+                    "⚠️ Warning: NSCameraUsageDescription is missing from Info.plist. Camera functionality in AdyenCardScanner SDK might be limited or fail."
```

Comment:
Should this be more explicit and mention release build crash?

Reply:
I can add that as a reason

Reply:
Think assert could be simpler, fails on debug and nothing on release

Reply:
Actually, Eren is right. `assertionFailure` achieves the same behavior. I changed it.